src/components/leaflet/index.js

```diff
@@ -93,7 +93,7 @@ var Leaflet = Base.extend({
 
   rotateMarker: function(e) {
     var heading = e.geoposition.coords.heading;
-    heading && this.marker && this.marker.setIconAngle(heading);
+    this.marker && this.marker.setIconAngle(heading);
```

Comment:
You are right, this was a bug!
The reason why this check was here is because there is no guaranteed `heading` property in a Coordinates object.

>  If speed is 0, heading is NaN. If the device is unable to provide heading information, this value is null.

from https://developer.mozilla.org/en-US/docs/Web/API/Coordinates

My implementation was buggy, but I think we should keep the check, just with a better implementation, like checking for NaN and null explicitly. What do you think?